src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -792,6 +794,30 @@
         </xsd:sequence>
     </xsd:complexType>
 
+
+    <xsd:complexType name="mailer_dkim_signer">
+        <xsd:sequence>
+            <xsd:element name="options" type="metadata" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
the name should be `option`, not `options`, as each tag represents a single option.